src/google_cloud_debugger/google_cloud_debugger_lib/dbg_breakpoint.h

```diff
@@ -152,6 +154,11 @@ class DbgBreakpoint : public StringStreamWrapper {
                             IEvalCoordinator *eval_coordinator,
                             IDbgObjectFactory *obj_factory);
 
+  // Evaluates expressions and returns the result in vector DbgObject.
```

Comment:
nit: does this return a vector result?  Looking above it looks like it populates a map in the class?